actionlist/__tests__/index.android.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Any place where you disable linting should have an additional comment explaining why it's disabled.